ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -359,6 +359,7 @@ function wait_machineconfig_applied() {
     echo "MCP ${role} status is ${mcp_status}"
     if [[ X"$mcp_status" != X"True" ]]; then
         echo "Timeout waiting for mcp updated"
+        oc describe mcp ${role}
```

Comment:
https://github.com/openshift/release/blob/aecef90e8e2bfb35c74fa0b1034d4d03cbbd2aba/ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh#L17-L18 does describe the mcp.